cluster/warewulf-cluster.spec.in

```diff
@@ -37,16 +36,14 @@ with Warewulf.
 %{__make} install DESTDIR=$RPM_BUILD_ROOT %{?mflags_install}
 
 
-%clean
-rm -rf $RPM_BUILD_ROOT
-
-
 %files
 %defattr(-, root, root)
-%doc AUTHORS COPYING ChangeLog INSTALL LICENSE NEWS README TODO
+%doc AUTHORS ChangeLog INSTALL NEWS README TODO
+%license LICENSE COPYING
```

Comment:
Where does `%license` put the file? `/usr/share/licenses` or something like that isn't it?

And `%license` isn't supported on RHEL 6 RPM is it? At least, I don't see a RPM macro from a quick search.

Reply:
I believe that's correct. It was implemented to make sure that license requirements would be met, even if the user turned off documentation.

I don't know if license is supported in EL6, but the spec file will fail in many other places on EL6. RHEL6.X is DOA this year and I don't test it on head node installs (does anyone?). I do test CentOS 6.X compute node creation under newer head nodes. The ability to bring up a CentOS6 subcluster when needed can be useful.

Reply:
RHEL 6.x is 2024, and 5.x is November according to RedHat's site.... For extended support anyway. I had a year or two ago a 6.x provisioner. The one I just recently build was 7.x only because other requirements (login node as well) otherwise I probably would have built a 6.x provisioner.